deploy/k8s/charts/trustification/values.yaml

```diff
@@ -212,4 +213,10 @@ modules:
     image: {}
     rust: {}
 
+  report:
+    enabled: true
+    ingress: {}
+    infrastructure: {}
+    image: {}
+    patternflyUrl: "https://unpkg.com/@patternfly/patternfly/patternfly.css"
```

Comment:
As said before, this isn't a viable option for many reasons. I think we should provide a suitable option by default and save us some trouble during productization.

Reply:
Can we define several CSS options by adding a new configuration file?

Reply:
IIRC we decided on moving away from this approach and add some static CSS?